CMake/ctkMacroBuildQtPlugin.cmake

```diff
@@ -88,7 +88,7 @@ macro(ctkMacroBuildQtPlugin)
   # Wrap
   set(MY_QRC_SRCS "")
   if(CTK_QT_VERSION VERSION_GREATER "4")
-    qt5_wrap_cpp(MY_MOC_CPP ${MY_MOC_SRCS} TARGET ${lib_name} OPTIONS -DHAVE_QT5)
+    qt5_wrap_cpp(MY_MOC_CPP ${MY_MOC_SRCS} OPTIONS -DHAVE_QT5)
```

Comment:
Indeed, Qt 5.1.0 does not support the TARGET argument. I added a condition on the Qt version.